analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ExpandingArchives.cs

```diff
@@ -35,6 +35,7 @@ public void ExtractEntry(ZipArchiveEntry entry)
             Stream stream;
 
             entry.ExtractToFile(""); // Noncompliant
+            entry?.ExtractToFile(""); // Noncompliant
```

Comment:
Is this change intended?

Reply:
Yes, it's the [Bump Coverage](https://github.com/SonarSource/sonar-dotnet/pull/5165/commits/319b0e759ccedb5d3240b3366af5270ed3a242b4) commit. `MemberBindingExpressionSyntax` case was not covered in the handler.